packages/contentstack-branch/src/config/index.ts

```diff
@@ -1,9 +1,8 @@
 const config = {
-  baseUrl: 'https://dev16-branches.csnonprod.com/v3/stacks/branches/compare',
+  baseUrl: 'https://dev16-api.csnonprod.com/v3/stacks/branches_compare',
```

Comment:
@aman19K @shafeeqd959 Why are we still using this config?
Can't we set the region and get this from the region config?

Reply:
@netrajpatel We aren't using the configuration file in the SDK integration PR.
PR:- https://github.com/contentstack/cli/pull/732